packages/server/src/post/service.ts

```diff
@@ -29,10 +32,18 @@ export class PostService extends Service {
     }
 
     try {
-      await axios.post(url, data, config)
+      await backOff(async () => axios.post(url, data, config), {
+        jitter: 'none', // TODO: We should enable to jitter if the post service may be called for the same resource on multiple nodes
```

Comment:
What it this comment about? I don't really understand what you're saying.

Reply:
If the post service, for some reason may call the same URL at the same time and on multiple instances of the messaging server, we should enable the jitter!